src/main/scala/ie/zalando/fabric/gateway/service/ResourcePersistenceValidations.scala

```diff
@@ -64,7 +64,7 @@ object ResourcePersistenceValidations {
       validationRequest.resource.paths.keys.toList.map(validateStarStarPathPosition).sequence,
       validateStackSetIntegration(validationRequest.hasExternallyManagedServices, validationRequest.definedServiceCount),
       validationRequest.corsConfig
-        .map(_.allowedOrigins.toList.map(hostname => validateCorsHostname(hostname)))
+        .map(_.allowedOrigins.toList.map(hostname => validateCorsUri(hostname)))
```

Comment:
I'd say naming-wise "hostname" should be "origin" and "validateCorsUri" should be "validateCorsOrigin"